src/html5/storage/webstorage/localstorage.spec.js

```diff
@@ -0,0 +1,83 @@
+import Promise from 'es6-promise';
```

Comment:
I think these changes don't pertain to this ticket, do they? About local/session storage and tests. And given the upcoming redesign, these would no longer be relevant.